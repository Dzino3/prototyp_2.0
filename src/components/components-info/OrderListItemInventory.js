import React from 'react'


function OrderListItemInventory({item}) {
  return (
    <div className='orderListItemInventory'>
        <div className='block'>
            <p className='p2'>Factory delivery date: </p>
            <p>{item.FDD}</p>
            <p className='p2'>ADD: </p>
            <p>{item.ADD}</p>
            <p className='p2'>Inv. date: </p>
            <p>{item.INV_DATE}</p>
        </div>
        <div className='block'>
            <p className='p2'>Billing date: </p>
            <p>{item.BILLING_DATE}</p>
            <p className='p2'>Billing number: </p>
            <p>{item.BILLING_NUMBER}</p>
        </div>
        <div className='block'>
            <p className='p2'>Days on Inv.: </p>
            <p>{item.DAYS_ON_INV}</p>
            <p className='p2'>Order status: </p>
            <p>{item.ORDER_STATUS}</p>
        </div>
        <div className='block'>
            <p className='p2'>Qty.: </p>
            <p>{item.QTY}</p>
        </div>
    </div>
  )
}

export default OrderListItemInventory
