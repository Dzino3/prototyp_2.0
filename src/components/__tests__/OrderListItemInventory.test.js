import React from "react";
import renderer from 'react-test-renderer';
import OrderListItemInventory from '../components-info/OrderListItemInventory';

describe(`Rendering OrderListItemInventory`, () => {

    test('should render OrderListItemInventory with item INVENTORY', function () {
        let itemX = {
            item: "INVENTORY"
        }
        let tree = renderer.create(<OrderListItemInventory item={itemX}></OrderListItemInventory>);

        expect(tree.toJSON()).toMatchSnapshot();
    });

    test('should render OrderListItemInventory with no item', function () {
        let itemX = {}
        let tree = renderer.create(<OrderListItemInventory item={itemX}></OrderListItemInventory>);

        expect(tree.toJSON()).toMatchSnapshot();
    });
})