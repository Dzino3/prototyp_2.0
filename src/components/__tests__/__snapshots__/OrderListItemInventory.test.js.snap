// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering OrderListItemInventory should render OrderListItemInventory with item INVENTORY 1`] = `
<div
  className="progressBarhelper"
>
  <div
    className="orderListItemInventory"
  >
    <div
      className="block"
    >
      <p
        className="p2"
      >
        listItemILabel1
         
      </p>
      <p />
      <p
        className="p2"
      >
        listItemILabel2
         
      </p>
      <p />
    </div>
    <div
      className="block"
    >
      <p
        className="p2"
      >
        listItemILabel4
         
      </p>
      <p />
      <p
        className="p2"
      >
        listItemILabel5
         
      </p>
      <p />
    </div>
    <div
      className="block"
    >
      <p
        className="p2"
      >
        listItemILabel6
         
      </p>
      <p />
      <p
        className="p2"
      >
        listItemILabel3
         
      </p>
      <p />
    </div>
    <div
      className="block"
    >
      <p
        className="p2"
      >
        listItemILabel8
         
      </p>
      <p />
    </div>
  </div>
  <div
    className="progressBarImgI"
  />
</div>
`;

exports[`Rendering OrderListItemInventory should render OrderListItemInventory with no item 1`] = `
<div
  className="progressBarhelper"
>
  <div
    className="orderListItemInventory"
  >
    <div
      className="block"
    >
      <p
        className="p2"
      >
        listItemILabel1
         
      </p>
      <p />
      <p
        className="p2"
      >
        listItemILabel2
         
      </p>
      <p />
    </div>
    <div
      className="block"
    >
      <p
        className="p2"
      >
        listItemILabel4
         
      </p>
      <p />
      <p
        className="p2"
      >
        listItemILabel5
         
      </p>
      <p />
    </div>
    <div
      className="block"
    >
      <p
        className="p2"
      >
        listItemILabel6
         
      </p>
      <p />
      <p
        className="p2"
      >
        listItemILabel3
         
      </p>
      <p />
    </div>
    <div
      className="block"
    >
      <p
        className="p2"
      >
        listItemILabel8
         
      </p>
      <p />
    </div>
  </div>
  <div
    className="progressBarImgI"
  />
</div>
`;
